conferences/abstracts/EuroPython2020-talk.md

```diff
@@ -0,0 +1,73 @@
+# EuroPython 2020 Proposal - Talk
+
+**Where:** Online! :woman_technologist:
+
+**When:** 23rd - 26th July 2020
+
+**Conference website:** <https://ep2020.europython.eu/>
+
+**Call for Proposals website:** <https://ep2020.europython.eu/call-for-proposals/>
+
+## Title
+
+Sharing Reproducible Python Environments with Binder
+
+## Subtitle
+
+(_Optional_)
+
+## Abstract (long)
+
+_At most 1500 characters. Current count (with spaces): 1182_
+
+As reproducibility gains traction in the data science community, the need to package code, data and the computational environment is growing.
+
+There are many tools that address different aspects of this type of packaging, such as Jupyter Notebooks for literate programming, Docker for containerising and porting computational environments, and so on.
+But they represent barriers to reproducibility as each one requires time and effort to learn.
+
+Project Binder provides the tooling for generating reproducible computational analyses with a web-based interface, such that users do not have to worry about all the moving parts so long as they have followed software best practices: their code is version controlled and they've captured the dependencies the analysis needs to run.
+Binder then allows the users to easily share their analyses by hosting the compute in the cloud and providing a unique URL to the code repository.
```

Comment:
Can you be clearer about 'users', perhaps naively I see two groups a) users who have a repo and Binderise (is that a word?) it b) users who interact with the hosted Binder instance

Reply:
"code creators"?

Reply:
Y